flink-connector-oracle-cdc/pom.xml

```diff
@@ -75,11 +75,19 @@ under the License.
             <version>${flink.version}</version>
             <scope>test</scope>
         </dependency>
+
         <dependency>
             <groupId>com.oracle.ojdbc</groupId>
             <artifactId>ojdbc8</artifactId>
             <version>19.3.0.0</version>
         </dependency>
+
+      <dependency>
+        <groupId>com.oracle.database.xml</groupId>
+        <artifactId>xdb</artifactId>
+        <version>19.3.0.0</version>
+      </dependency>
```

Comment:
I think we need to add this dependency in `flink-sql-connector-oracle-cdc` pom file to build the sql fat jar.

Reply:
Ok, I will move the dependency  to `flink-sql-connector-oracle-cdc` and add a test.

Reply:
we don't need move the dependency, we just need add a <include> line in flink-sql-connector-oracle-cdc

Reply:
ok